### PR TITLE
harshaparida's card

### PR DESCRIPTION
**Pull request by harshaparida**
I have added my card in this index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,6 +121,36 @@ <h2>A project for first-time contributions</h2>
...
+            <a href="https://www.github.com/harshaparida" target="_blank">harshaparida</a>
+          </p>
+          <p class="about">
+            Hi! I'm a Student developer, and Machine Learning enthusiast. New to open-source. Feel free to contact me!
+          </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.w3schools.com/https://www.w3schools.com/" target="_blank" title="First Resource">W3Schools</a>
```
Comment (theAravinthM) [inline index.html:140]: The hyper link seems to be pasted twice inside the `href` attribute. Kindly rectify that :)

**Review by theAravinthM — CHANGES_REQUESTED** [review]
Hi @harshaparida Everything looks great ! Only one change required in your commit to get your PR merged. Have provided the comment to help you rectify. Get back with the changes to get your PR merged soon :)

File: index.html (hunk 1 above)
Comment (harshaparida) [reply index.html]: Thank you for letting me know. I have rectified the hyperlink.
edit

**Comment by theAravinthM** [comment]
Cheers on your PR Merge 🥂, and Thanks for your Contribution! Your card will be live on our Website in a moment!

Now you can follow the Next Steps. You are also welcome to join our Discord Server: https://discord.gg/tWkvS4ueVF and be a part of our community.

Happy contributing!

**Pull request merged**